validation/src/test/scala/hmda/validation/rules/lar/validity/V335Spec.scala

```diff
@@ -0,0 +1,36 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V335Spec extends LarEditCheckSpec {
+  property("Succeeds when property type is not 3") {
+    forAll(larGen) { lar =>
+      whenever(lar.loan.propertyType != 3) {
```

Comment:
The validation should not pass for all LARs.  If the property type is 3, and the income is numeric (or, for that matter, "nonsense_string" or the like), then it should fail.
(But if the property type is not 3, then it should pass no matter what.  And those are the cases covered by this test.)

This answer may not have addressed your question.  If not, feel free to ping me in chat (or here, if you prefer).  Or if you haven't already (which you may have), try removing the line and see whether/how/why it fails.  And definitely feel free to ask if you get stuck on the "why" part, as that may be part of what you're asking in the first place, and my goal is not to make you go around in circles. :)